client/routes/Navigation.jsx

```diff
@@ -36,12 +36,12 @@ const Navigation = () => {
             </Button>
             {menuExpanded && (
                 <Menu id="nav" className="nav" stackable>
-                    <Menu.Item>
+                    <Menu.Item className="logo">
                         <NavLink to="/">Teacher Moments</NavLink>
                     </Menu.Item>
 
                     <Menu.Menu>
-                        <Dropdown simple item text="Curated">
+                        <Dropdown simple item text="Moments">
```

Comment:
This drop down is supposed to move under the item presently called "Teacher Moments", but we were waiting until we had a final word on the corner logo. This menu item needs to merge with that item, not be renamed.

Reply:
sgtm